src/workspace/shared/usePortalManager.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
```

Comment:
This is also present in the /src/shared directory.

Reply:
Updated.